generator/feature_builder.cpp

```diff
@@ -408,6 +408,25 @@ void FeatureBuilder1::Serialize(TBuffer & data) const
 #endif
 }
 
+void FeatureBuilder1::SerializeBorder(serial::GeometryCodingParams const & params,
+                                      TBuffer & data) const
+{
+  data.clear();
+
+  PushBackByteSink<TBuffer> sink(data);
+  WriteToSink(sink, GetMostGenericOsmId().EncodedId());
+
+  CHECK_GREATER(m_polygons.size(), 0, ());
+
+  WriteToSink(sink, m_polygons.size() - 1);
+  for (auto const & polygon : m_polygons)
+  {
+    WriteToSink(sink, polygon.size());
+    for (auto const & p : polygon)
+      serial::SavePoint(sink, p, params);
```

Comment:
I strongly suggest to get rid of serial::SavePoint here. Use coding::EncodePointDelta instead.

Reply:
done